pkg/fqdn/name_manager_test.go

```diff
@@ -147,3 +148,46 @@ func TestNameManagerMultiIPUpdate(t *testing.T) {
 	nameManager.Unlock()
 
 }
+
+func TestActiveConnectionsTTLUpdate(t *testing.T) {
+	now := time.Now()
+	dnsZombies := NewDNSZombieMappings(defaults.ToFQDNsMaxDeferredConnectionDeletes, defaults.ToFQDNsMaxIPsPerHost)
+	sharedIP := netip.MustParseAddr("1.1.1.1")
+	dnsZombies.Upsert(now, sharedIP, "cilium.io")
+
+	var (
+		activeConnectionTTL = 2
+		nameManager         = NewNameManager(Config{
+			MinTTL:               1,
```

Comment:
nit: If the times are seconds, then they should probably be declared as so. This will avoid misuse later on, and save some casting:

```suggestion
		activeConnectionTTL = 2 * time.Second
		nameManager         = NewNameManager(Config{
			MinTTL:               1 * time.Second,
```